ast/Trees.h

```diff
@@ -675,6 +675,11 @@ class EmptyTree final : public Expression {
  * Desugar string concatenation into series of .to_s calls and string concatenations
  */
 
+// TODO Find a better place for this
+bool classDefinesBehavior(std::unique_ptr<ast::ClassDef> &);
```

Comment:
can we drop a fat comment explaining what does "defineBehavior" mean in english?